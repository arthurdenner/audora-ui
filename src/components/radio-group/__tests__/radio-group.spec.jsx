import React from 'react';
import { render } from 'enzyme';
import RadioGroup from '../radio-group';

describe('RadioGroup', () => {
  it('renders correctly', () => {
    const items = [
      { name: 'color', value: 'red', label: 'Red' },
      { name: 'color', value: 'blue', label: 'Blue' },
      { name: 'color', value: 'yellow', label: 'Yellow' },
      { name: 'color', value: 'green', label: 'Green' },
    ];
    const wrapper = render(<RadioGroup items={items} />);
    expect(wrapper).toMatchSnapshot();
  });
});
