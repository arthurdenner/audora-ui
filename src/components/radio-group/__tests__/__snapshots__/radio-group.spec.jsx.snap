// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioGroup renders correctly 1`] = `
<div
  class="sc-htpNat dePyzG"
>
  <label
    class="sc-bdVaJa hKAetY"
  >
    <input
      name="color"
      type="radio"
      value="red"
    />
    <span
      class="sc-bwzfXH drzoWa"
    >
      Red
    </span>
  </label>
  <label
    class="sc-bdVaJa hKAetY"
  >
    <input
      name="color"
      type="radio"
      value="blue"
    />
    <span
      class="sc-bwzfXH drzoWa"
    >
      Blue
    </span>
  </label>
  <label
    class="sc-bdVaJa hKAetY"
  >
    <input
      name="color"
      type="radio"
      value="yellow"
    />
    <span
      class="sc-bwzfXH drzoWa"
    >
      Yellow
    </span>
  </label>
  <label
    class="sc-bdVaJa hKAetY"
  >
    <input
      name="color"
      type="radio"
      value="green"
    />
    <span
      class="sc-bwzfXH drzoWa"
    >
      Green
    </span>
  </label>
</div>
`;
